src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -909,6 +909,9 @@ export default {
 
         case FilingTypes.INCORPORATION_APPLICATION:
           try {
+            // show spinner since the network calls below can take a few seconds
+            this.$root.$emit('showSpinner', true)
```

Comment:
I decided that this is the only place at the moment that requires a "busy spinner". All other redirections happen pretty much right away.